src/components/profile-view/profile-view.jsx

```diff
@@ -4,20 +4,19 @@ import {UserInfo} from "./user-info";
 import {FavoriteMovies} from "./favorite-movies";
 import {UpdateUser} from "./update-user";
 import {Row, Container, Col} from "react-bootstrap";
+import {useSelector, useDispatch} from "react-redux";
+
+export const ProfileView = ({removeFavorite, handleLogout}) => {
+  const user = useSelector((state) => state.user);
+  const token = useSelector((state) => state.token);
```

Comment:
B. `state.token` returns an object, not a string anymore. Or you can access it like this(ugly):

`const token = useSelector((state) => state.token.token);`